src/components/CheckboxInput/CheckboxInput.module.css

```
@@ -42,7 +42,7 @@
 }
 
 /**
- * The visible checkbox svg icon element.
+ * 1) The visible checkbox svg icon element.
```

We don't need the number here.
```suggestion
 * The visible checkbox svg icon element.
```

---

removed thanks!